DEPRECATED.md

```diff
@@ -19,6 +19,11 @@ The following features have been DEPRECATED and will be removed in the specified
 * `value` and `regex` fields in the `HeaderMatcher` message is deprecated. Use the `exact_match`
   or `regex_match` oneof instead.
 
+
+* Admin mutations should be sent as POSTs rather than GETs. Starting in 1.7, these will result
```

Comment:
Please make a new 1.7.0 version section above the 1.6.0 version and add this.

Reply:
OK.  I thought that we were declaring it deprecated effectively now.  In 1.7 it would be *beyond* deprecated, so to speak.

Reply:
It is deprecated now, made official in 1.7, and then we can make the switch right after 1.7 is released.